test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -1485,26 +1485,30 @@ BOOST_AUTO_TEST_CASE(disallow_declaration_of_void_type)
 	CHECK_ERROR(sourceCode, TypeError, "Not enough components (0) in value to assign all variables (1).");
 }
 
-BOOST_AUTO_TEST_CASE(overflow_caused_by_ether_units)
+BOOST_AUTO_TEST_CASE(no_overflow_with_large_literal)
```

Comment:
Can you somehow make sure that this an the next test end up next to each other or at least have a similar name or reference each other in a comment?

Reply:
I think this is the only exception, the others are grouped. Not sure what is the best name for grouping, perhaps `overflow_not_caused_with_large_literal_without_denomination`.